### DISCUSSION TITLE
Github copilot blocks auto complate

### DISCUSSION THREAD

**Discussion by hsnbozkurt** [community/community#11410] [Copilot Conversations]
when github copilot makes a inline suggestion the autocomplate not shows up this is important becuse copilot sometimes suggests the wrong property

**Comment by drizztdourden08** [comment]
Same thing happens for me. this is very annoying at times to not have access to intellisense when it is activated and showing a suggestion.

Also, Hover operations do not work either (For example to see the structure of a functions to know what should be in it)

**Reply by JasonJooste** [reply]
I'm also having the same problem

**Reply by Attachment-Studios** [reply]
Doesn't VS CODE have a "Forced Suggestions" feature, `Alt + Tab` and will show you some suggestions... Maybe that helps...? I mean I feel great that suggestions and copilot text doesn't interfere much but this might not be true for others so maybe they can add such features like enable/disable normal suggestions for either copilot or vs code...
